how-to/features/tls-vault.rst

```diff
@@ -0,0 +1,69 @@
+TLS Vault
+=========
+
+This feature is used to encrypt all cloud service endpoints (both public
```

Comment:
It would help to be more explicit: "TLS Vaut is used to..."

Reply:
SInce this is under TLS Vault, I do not believe reusing TLS Vault below is needed